### DISCUSSION TITLE
docs.obsidianweb.net continues offline

### DISCUSSION THREAD

**Discussion by vamonk** [KosmosisDire/obsidian-webpage-export#535] [General]
Hello, 

the documentation is not available, is this a sign that the project is not continuing?

Error 1033 appears.

Best regards
Volker

**Comment by KosmosisDire** [comment]
Sorry, I am hosting it on my own web server and the power had gone out. I have brought it back up but it is not appearing. I will get it working again when I am not so busy. I also may switch to a more reliable hosting method. Thanks!
